### PR TITLE
inmemorypipe accepts pandas df

### PR DESCRIPTION
**Pull request by navinrathore**
**inMemoryPipe** can be created using either Python **DataFrame** or **Pandas DataFrame**

### CODE REVIEW

**Comment by navinrathore** [comment]
* fixes a part of https://github.com/zinggAI/zingg/issues/347

File: python/zingg/pipes/pipes.py [hunk 1]
```diff
@@ -48,9 +48,18 @@ def setDbTable(self, dbtable):
...
 
 class InMemoryPipe(Pipe):
-    def __init__(self, name):
+    def __init__(self, name, df = None):
         Pipe.__init__(self, name, Format.INMEMORY.type())
-    def setDataset(self, ds):
-        Pipe.getPipe(self).setDataset(ds)
+        if (df is not None):
+            self.setDataset(df)
+    def setDataset(self, df):
```
Comment (sonalgoyal) [inline python/zingg/pipes/pipes.py:55]: line break should be there between methods in python

**Review by sonalgoyal — CHANGES_REQUESTED** [review]
please add a unit test case

File: python/zingg/pipes/pipes.py (hunk 1 above)
Comment (navinrathore) [reply python/zingg/pipes/pipes.py:69]: added

**Comment by navinrathore** [comment]
> please add a unit test case
Test case shall be taken care of a separate issue.

**Pull request merged**